modules/installation-guide/partials/proc_installing-che-on-codeready-containers-using-chectl.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// installing-{prod-id-short}-on-codeready-containers
+
+[id="installing-{prod-id-short}-on-codeready-containers-using-{prod-cli}_{context}"]
+= Installing {prod-short} on CodeReady Containers using {prod-cli}
+
+This section describes how to install {prod-short} on CodeReady Containers using {prod-cli}.
```

Comment:
Not always avoidable, but I try not to start modules with just a "This section describes how to do [module title]" type of opening. Perhaps describe when/why this can be useful/needed/important (aka the "user story")?